qiskit_ibm_runtime/options/twirling_options.py

```diff
@@ -32,8 +32,8 @@ class TwirlingOptions:
     Args:
         enable_gates: Whether to apply 2-qubit gate twirling. Default: False.
 
-        enable_measure: Whether to enable twirling of expectation value measurements
-          in Estimator. Default: True.
+        enable_measure: Whether to enable twirling of measurements. The twirling can only be applied to
```

Comment:
slight clarification:

```suggestion
        enable_measure: Whether to enable twirling of measurements. Twirling will only be applied to
```